lib/apis/InSkillPurchase.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const _ = require('lodash');
+const rp = require('request-promise');
+
+class InSkillPurchase {
+  constructor(alexaEvent) {
+    this.alexaEvent = alexaEvent;
+  }
+
+  isIspAllowed() {
```

Comment:
Just a naming thing (suggestion), I would name this method `isAllowed` instead of `isIspAllowed ` since the class already has the name on it (`InSkillPurchase`). It is known that all methods inside that class are related to ISP so probably you should change the name of the method.